api/proto-go/all.go

```diff
@@ -1,17 +1,3 @@
-// Copyright (c) 2021 Terminus, Inc.
```

Comment:
do not commit these changes. too many irrelevant files.
or commit in another pr.

Reply:
done